src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/OffenderEventSubscriber.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.services
+
+import com.google.gson.Gson
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.jms.annotation.JmsListener
+import org.springframework.stereotype.Service
+
+data class OffenderMovementReception(
+  val eventType: String,
+  val offenderIdDisplay: String,
+)
+
+data class Event(val Message: String)
+
+@Service
+class OffenderEventSubscriber(
```

Comment:
new subscriber which will replace old one eventually.  At present both will listen on the events, which means that the second one will be a no-op as the patient will have been deleted already